### ISSUE TITLE
Scrolling not working on Chrome for Android Beta (v35)

### ISSUE THREAD

**Issue by mydea** [hammerjs/hammer.js#541]
I use Hammer.js for horizontal gestures on list elements - the user can swipe left or right on list elements for additional options.

While I got this to work on Chrome for Android (v34) and on the Stock Browser on a Galaxy S2, I can't scroll on Chrome for Android Beta (v35). While the swipe gestures are working correctly, scrolling is not possible.

I have the following configuration (copy & pasted it together from various posts I found about this topic):

swipe_velocity: 0.2,
swipeVelocityX: 0.2,
swipe_time: 300,
dragLockToAxis: true,
dragBlockHorizontal: true,
drag: true,
drag_block_horizontal: true,
drag_lock_min_distance: 20,
hold: false,
release: true,
tap: false,
touch: true,
transform: false

I don't know if it is a bug in Chrome or another problem.

**Comment by charandas** [comment]
@RyanMG thanks for suggesting the workaround above. I tried it but doesn't seem to work for me. I seem to be in little different situation though. Touch scrolling breaks for me, but not 100% of time, its more like 85-90% of the time. It resumes if I open devtools or move the mouse cursor.

Of course, Chrome versions have moved on since you chatted here about this. I am using 46. Did you find anything more conclusive about this issue?

I am using hammer 2.0 to disable some events on the app global container, namely:
`transform dragleft dragright swipe swipeleft swiperight`

On top of it, I am using angular. So these elements are in an `ng-repeat`. Once the scroll breaks, if I don't resort to the mouse, the scrollbars do work with touch, but that's medieval. Why would pan-to-scroll be so difficult, I wonder. Anyway, cannot reproduce this issue with Canary.

Tried several chromium builds, thinking its this [fix](https://code.google.com/p/chromium/issues/detail?id=417345) that made a difference, but haven't been able to prove that to myself.

**Comment by RByers** [comment]
Yes @majido has fixed at least one problem like that recently.  

>  Weird, but what can you do.

As I mentioned above, you can filed a bug with repro steps and we'll get it fixed in the next Chrome release!  If you can still reproduce an issue on Chrome 47+ (especially if removing hammer makes no difference) then PLEASE [file a bug](http://crbug.com/new) with repro steps and we will get it fixed ASAP.

>  For what seems like such a common use case on the mobile web, it's a pretty silly bug.

This stuff is complicated (and a little brittle) because of our threaded scrolling architecture where we want handle touch scrolling completely in our compositor without blocking on the blink thread (which is the only thread that knows about the DOM in any detail).  Sorry for the pain this is causing!

**Comment by charandas** [comment]
Thanks @RByers for the detailed response.

I cannot reproduce my issues with Chrome Canary 48.0.2547.0, but downloading chromium builds until 48.0.2527.0 hasn't given me the same successful results.

I was hoping I could pin it to a commit, so I can watch out for it in upcoming chrome releases, but haven't been able to in my brief testing. Which commit by @majido are you mentioning about - I am curious?

I also downloaded Sawbuck and ran with chrome and chrome frame set to verbose. But when the locking freezes, or in general, I don't get any messages from the renderer. Perhaps I am missing something.

**Comment by RByers** [comment]
> I was hoping I could pin it to a commit, so I can watch out for it in upcoming chrome releases, but haven't been able to in my brief testing. Which commit by @majido are you mentioning about - I am curious?

Just [the bug you found](https://code.google.com/p/chromium/issues/detail?id=417345).  I took a quick look through the other changes I was thinking about, and they all seem unlikely to be your issue.  I'd suggest trying [Chrome Beta](https://www.google.com/chrome/browser/beta.html), if you don't see any problems there then it's likely to be fixed with Majid's change.

**Comment by charandas** [comment]
@RByers yes, I don't see the problem in the beta version. Do you know when the next release of chrome is going to be out, in other words, when will this beta be promoted to stable?

**Comment by RByers** [comment]
Chrome is released about every 6 weeks, and Chrome 46 went stable on Oct 13.  So I'd guess early December 47 should get promoted to stable.
